### PR TITLE
Category page

### PR DESCRIPTION
**Pull request by Kwolik**
- Create Category page
- Add navigation in button Start
- Download category list from api

### CODE REVIEW

**Review by YELOYELO99 — APPROVED** [review]

**Review by KamilDonda — APPROVED** [review]

File: screens/Category.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      marginTop: "18%",
+      height: "52%",
+    },
+    name: {
+      color: "#2D2D2D",
+      fontSize: 18,
+    },
+  });
+
+  const [category, SetCategory] = React.useState([]);
```
Comment (michal-lepik) [inline screens/Category.js]:
```suggestion
  const [category, setCategory] = React.useState([]);
```
These should be camelCase

File: screens/Category.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    const responseJson = await response.json();
+
+    if (responseJson.trivia_categories) {
+      SetCategory(responseJson.trivia_categories);
+    }
+  };
+
+  React.useEffect(() => {
+    getMovieRequest();
+  }, []);
```
Comment (michal-lepik) [inline screens/Category.js:79]: I think the `getMovieRequest` should be included in the dependency array, isn't eslint complaining about this one?

File: Navigation.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import AboutScreen from "./screens/About";
+import HistoryScreen from "./screens/History";
+import CategoryScreen from "./screens/Category";
+
+export default function Navigation() {
+  const Stack = createStackNavigator();
+
+  return (
+    <NavigationContainer>
+      <Stack.Navigator headerMode={"none"} initialRouteName={"Home"}>
```
Comment (michal-lepik) [inline Navigation.js]:
```suggestion
      <Stack.Navigator headerMode="none" initialRouteName="Home">
```
Curly braces are not necessary for string props 😛

Comment (michal-lepik) [reply Navigation.js]: This applies to any props wrapped like that actually :suspect:

**Review by michal-lepik — APPROVED** [review]

File: screens/Category.js (hunk 2 above)
Comment (Kwolik) [reply screens/Category.js:79]: eslint shows no errors.  I think you don't need to enter into the dependency array, you only need to download all available categories once.

**Pull request merged**